src/main/java/com/puppycrawl/tools/checkstyle/utils/TokenUtil.java

```diff
@@ -291,6 +291,60 @@ public static boolean isOfType(int type, int... types) {
         return matching;
     }
 
+    /**
+     * Determines if the token type belongs to the given first parameter.
+     *
+     * @param type the Token Type to check
+     * @param first the acceptable token
+     *
+     * @return type is equals to first
+     */
+    public static boolean isOfType(int type, int first) {
+        return type == first;
+    }
+
+    /**
+     * Determines if the token type belongs to the given first or second parameter.
+     *
+     * @param type the Token Type to check
+     * @param first the acceptable token
+     * @param second the acceptable token
+     *
+     * @return type if it is equals to first or second
```

Comment:
`type if it is equals to first or second`
--->
`true if type equals to first or second`
Please fix `@return` description in other methods too.

Reply:
Changes Done